json-logic/logic.md

```diff
@@ -0,0 +1,23 @@
+# Mapping generic logic to JSON-Logic
+
+This files summarizes how logic operations can be mapped to JSON-Logic.
```

Comment:
I had a deeper look into JSON logic yesterday and today. I hate to say it, but JSON logic is not complete, not well maintained and **very JS specific**. 

I propose the following restriction that we allow only a subset of JSON Logic features to keep interoperability with other platforms easy. 
That means: 
* We don't allow truthy or falsy values, only true or false. This means boolean operators on non-boolean values are not supported. 
* I would suggest removing map, reduce, filter, merge, log from supported operations.

Note: We can still use JSON Logic everywhere. This is just about restricting things that are likely to break outside of the JS ecosystem.

Reply:
What about string operations? Should we restrict them too? At least the names sound a lot like JS, too. And I don't think there is any requirement for string operations at the moment.

Reply:
The string operations are supported in all serious languages out of the box, so I assumed any JSON-Logic client will simply them. We should keep the same name, so people can just use existing JSON-Logic clients (available in python, php, js, ruby, go, java, swift and more).

My big problem is that the view on requirements for the evaluation are incomplete, so this is more or less an educated guess.